api/index.js

```diff
@@ -2,30 +2,31 @@ const url = 'https://api.telegram.org/bot';
 const apiToken = process.env.BOT_TOKEN;
 const axios = require('axios');
 const crypto = require('crypto');
+const space = require('./spacifier');
```

Comment:
I think it's misspelled (see reference in line 11)
```suggestion
const spacify = require('./spacifier');
```

Reply:
Oh thanks for pointing out!